python_modules/dagster/dagster/_core/errors.py

```diff
@@ -684,5 +684,5 @@ class DagsterDefinitionChangedDeserializationError(DagsterError):
     """
 
 
-class DagsterPipesExecutionError(DagsterError):
+class DagsterPipesExecutionError(Exception):
```

Comment:
Perhaps could be a `DagsterUserCodeExecutionError`, but there's no underlying error to wrap in this case

Reply:
leave a comment as to why this does not subclass like the other exceptions around it